pad.js

```diff
@@ -0,0 +1,13 @@
+pry = require('pryjs')
+
+var pad = Array.prototype.pad = function(min, pad) {
+  var array = this;
+
+  if (array.length <= min) {
+    return array
+  } else {
+    array.fill(pad)
+  }
+}
+
+exports.pad = pad;
```

Comment:
pad isn't quite working yet. the return value isn't what is specified in the requirements. keep working on this one 👍